executor/kava/guide.md

```diff
@@ -0,0 +1,221 @@
+```
+
+Query the new swap on bnbchain by its ID
+```bash
+  bnbcli token query-swap \
+  --swap-id ${BNBCHAIN_SWAP_ID} \
+  --chain-id Binance-Chain-Tigris
+```
+
+The deputy process will create an atomic swap on kava with the same information after _n_ blocks. It will log the expected kava swap ID as `swap.OtherChainSwapId` (it will be different than the bnbchain swap ID):
+```bash
+swap.OtherChainSwapId: edaf7deaf96d0ea583fa8a1cf3b547089418ad1065b4bca6ed856fcf8aaa110e
+```
+
+You'll need the kava swap ID and random number from above in order to claim the swap on kava.
+
+## Kava
+
+Claim atomic swap
+```bash
+kvcli tx bep3 claim ${KAVA_SWAP_ID} ${RANDOM_NUMBER} --from user
+# enter password
+```
+
+A successful claim should output a tx result log similar to:
+```bash
+logs:
+- msgindex: 0
+  success: true
+  log: ""
+  events:
+  - type: claimAtomicSwap
+    attributes:
+    - key: claim_sender
+      value: kava1xy7hrjy9r0algz9w3gzm8u6mrpq97kwta747gj
+    - key: recipient
+      value: kava1xy7hrjy9r0algz9w3gzm8u6mrpq97kwta747gj
+    - key: atomic_swap_id
+      value: 53ab009facd0a9b2e02971cd4228d969f2f4fc41712b6984975d42b655159b7e
+    - key: random_number_hash
+      value: 36371b5a6793cf411c1e55bbb5b6104981d79cce402259569f18bd044fb07303
+    - key: random_number
+      value: 90a62df3efb640ea361008f619adf878e88e72bb877a5aba62230c2c3bb2c94f
+  - type: message
+    attributes:
+    - key: action
+      value: claimAtomicSwap
+    - key: sender
+      value: kava1eyugkwc74zejgwdwl7mvm7pad4hzdnka4wmdmu
+    - key: module
+      value: bep3
+    - key: sender
+      value: kava1xy7hrjy9r0algz9w3gzm8u6mrpq97kwta747gj
+  - type: transfer
+    attributes:
+    - key: recipient
+      value: kava1xy7hrjy9r0algz9w3gzm8u6mrpq97kwta747gj
+    - key: amount
+      value: 29999000bnb
+```
+
+Funds have been transferred from the user's bnbchain address to the user's kava address. After _n_ blocks the deputy will relay the successful claim to bnbchain, closing the swap.
+
+# Transfer bnb from Kava to Bnbchain
+
+## Kava
+Create a new Atomic Swap
+```bash
+kvcli tx bep3 create ${KAVA_DEPUTY_ADDRESS} ${BNBCHAIN_USER_ADDRESS} ${BNBCHAIN_DEPUTY_ADDRESS} now 1111111bnb 1111111bnb 360 true --from user
+# Note: you may need to include `--chain-id testing`
```

Comment:
```suggestion
```

See above, should be able to remove.

Reply:
https://github.com/Kava-Labs/bep3-deputy/pull/1/commits/badeb0de0be59aea80df80664c5f2efce6b796fa